server/tests/vote.test.js

```diff
@@ -136,7 +136,7 @@ describe('## Vote APIs', () => {
         .expect(httpStatus.OK)
         .then((res) => {
           const vote = res.body;
-          expect(vote.postId).to.eql(`${postId}`);
+          expect(vote.entityId).to.eql(`${postId}`);
```

Comment:
These tests need to pass with `postId` to preserve the current clients.

Reply:
This part makes sense, I was only checking the web client and it didn't really break anything but I did consider filing in both postId and entityId in the vote model. I think this would be a decent solution for now and then I can have these tests still pass  (while adding new tests for general votes).